packages/metadata-service/src/readers/labirint.service/index.ts

```diff
@@ -0,0 +1,81 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import axios from 'axios';
+import { Prisma } from '@prisma/client';
+import cheerio from 'cheerio';
+import { Bundle } from '../../types';
+
+const READER_ID = 'LABIRINT';
+
+@Injectable()
+export class LabirintService {
+  private readonly rootUrl = 'https://www.labirint.ru';
+  private readonly rootUrlSearch = 'https://www.labirint.ru/search/';
+
+  async readData(identifier: string) {
+    try {
+      const firstResponse = await axios.get(
+        `${this.rootUrlSearch}${identifier}/?stype=0`
+      );
+      const $ = cheerio.load(firstResponse.data, null, false);
+      const id = $('.product-title-link').attr('href');
+      const secondResponse = await axios.get(`${this.rootUrl}${id}`);
+      return secondResponse.data;
+    } catch (e) {
+      throw new BadRequestException(e.message);
+    }
+  }
+
+  parseData(result): Bundle {
```

Comment:
better will make this method is private, because you use his only in this class(Encapsulation)